README.md

```diff
@@ -100,6 +77,50 @@ And learn how the API can be used to:
 * Predict susceptible, infected and removed population
 * Calculate confidence intervals of the predictions
 
+## Development Setup
+
+Open-SIR Development Environment uses
+[Pipenv](https://pipenv.pypa.io/en/latest/) to automatically create a virtual
+environment and manage python packages. The python packages required by
+Open-SIR are listed in the [Pipfile](Pipfile).
+
+After cloning the repository, change the current directory to the repository
+via `cd open-sir` and automatically install the environment from the Pipfile
+using Pipenv:
+```
```

Comment:
GitHub suggestions get confused when I suggest adding an extra line here, so I can't suggest it that way haha. Can you please add a blank line between scripts and what surrounds them please?

Reply:
sure!